joomla/joomla-cms
# [6.0] Remove leftover reCaptcha files

### Summary of Changes
Remove leftover reCaptcha files, forgoten in https://github.com/joomla/joomla-cms/pull/41530


### Testing Instructions

Code review.

Or build the release packge, and check for `/media/plg_captcha_recaptcha/` and `/media/plg_captcha_recaptcha_invisible/`. These folder should not exists.




### Actual result BEFORE applying this Pull Request
recaptcha files in release packge


### Expected result AFTER applying this Pull Request
no recaptcha files in release packge


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/182
